sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Generated/Models/ServerSku.cs

```diff
@@ -46,38 +46,26 @@ public partial class ServerSku
         private IDictionary<string, BinaryData> _serializedAdditionalRawData;
 
         /// <summary> Initializes a new instance of <see cref="ServerSku"/>. </summary>
-        /// <param name="name"> The name of the sku, typically, tier + family + cores, e.g. Standard_D4s_v3. </param>
-        /// <param name="tier"> The tier of the particular SKU, e.g. Burstable. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="name"/> is null. </exception>
-        internal ServerSku(string name, PostgreSqlFlexibleServerSkuTier tier)
```

Comment:
This is also a generic name. But if it's already published in an official release, then let's keep it as-is.

Reply:
It was first published last preview release, updated name for this official release.